docs/centralized-authentication-and-sso.md

```diff
@@ -0,0 +1,86 @@
+
+### 3 - Set up the synchronization
+
+Now we can continue to set up the synchronization. In your organization, go to _Settings > Synchronizations_ and press the 
+"Add Synchronization" button. Follow the steps to complete the configuration.
+
+![Create synchronization](/Resources/public/img/docs/articles/centralized-authentication-and-sso/configure-synchronization.png)
+
+Once the synchronization is set up, Private Packagist will retrieve your users and their permissions from the code hosting platform. 
+Private Packagist now knows which users have access to your organization and which repositories and teams/groups they can access 
+once they log in via OAuth. Only users who have logged into Private Packagist using OAuth will be considered active and count for billing purposes.
+
+Whether or not you are using automatic user provisioning, Private Packagist will always manage the same users as those provisioned 
+in your code hosting platform. If a user is removed, that same user will also be removed from Private Packagist.
+
+If you want to control which of your users should be allowed to access Private Packagist at all, you can opt to deactivate 
+users discovered by the synchronization by default. Then, only users that you activate manually will be able to log in. 
+To enable this behavior, go to the _Teams_ tab, and click on "View and manage all members of this organization" and check the box:
+
+![Deactivate users discovered by sync](/Resources/public/img/docs/articles/centralized-authentication-and-sso/deactivate-sync-members.png)
+
+### 4 - Enforce OAuth authentication
+
+Now we can enforce OAuth authentication for all users in the organization, to ensure that users must authenticate with your 
+identity system configured on the code hosting platform in order to access Private Packagist and cannot use an alternative 
+method to log into Private Packagist.
+
+Go to _Settings > Access_ and apply the changes. From now on, everyone - including yourself - will be required to 
+authenticate via your configured integration. 
+
+![Enforce OAuth authentication](/Resources/public/img/docs/articles/centralized-authentication-and-sso/enforce-oauth.png)
+
+### 5 - Share the organization-specific login link
+
+If you set up your own integration in Step 1, the final step is sharing your organization-specific login link with your users. 
+Look for the "Login link" button in _Settings > Integrations_ page.
+
+![Login link settings](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-settings.png)
+
+This link presents the option to authenticate with your configured integration. Once the user clicks on the
+"Login with &lt;integration name&gt;" button, the authentication process is initiated and your users are redirected to your 
+code-hosting platform to log in. . 
+
+![Login link example](/Resources/public/img/docs/articles/centralized-authentication-and-sso/login-link-example.png)
```

Comment:
The link says "Log in with Self-Hosted Gitlab" but the screenshot before says "GitLab" is this a bug or issue with the screenshot?

Reply:
Weird. That's not what I'm seeing here: 

![CleanShot 2024-11-26 at 13 56 02](https://github.com/user-attachments/assets/30ea93af-b94d-4dfe-b987-2dc0151af01e)

Reply:
The text in the red circle is different. One says `GitLab` the other `Gitlab`

Reply:
Oh! I thought you meant "self-hosted" text was missing. Screenshot must be from two different attempts. Thanks!